src/components/Sidebar/index.tsx

```diff
@@ -1,9 +1,19 @@
 import { Link } from '../Link';
-import { LinksSection, SidebarContainer } from './styles';
+import { LinksSection, SidebarContainer, Overlay, SidebarVolume } from './styles';
+import Modal from '~/components/Modal';
+import { useSidebarContext } from '~/context/useSidebarState';
+import { useMinWidth } from '~/hooks/useMinWidth';
+import { Breakpoint } from '~/styles/variables';
 
-export function Sidebar() {
+export interface InnerSidebarProps {
+  collapsed: boolean;
```

Comment:
Seguir o padrão do projeto e adicionar o prefixo `is` para booleanos

Reply:
comitado